DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,20 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>  
+  <class name="pat::PATTauDiscriminator"  ClassVersion="3">
+   <version ClassVersion="3" checksum="2916098408"/>  
+  </class>  
```

Comment:
```
> Ok -- if've done as advised. PATTauDiscriminator now has a class version
> 3. The PATTauDiscriminatorBase is a templated class.
```

Ah, I missed that about PATTauDiscriminatorBase.
I can't recall now if ROOT will even recognize the version for a class derived from a templated class

@Dr15Jones  please advise